README.md

```diff
@@ -33,12 +33,28 @@ Instead, create your own new node project, and install the oss-mariner package v
    1.1. <https://github.com/indeedeng/Mariner/blob/master/examples/runFasterCode.ts>
 1. Comment out the existing line that imports mariner.
 1. Uncomment the line saying how mariner would normally be imported.
+1. Create a config.json file inside examples folder
```

Comment:
I think these 2 bullets could be combined: "Create a config.json file inside the examples folder, with contents similar to this:"